src/components/Tabs/Tabs.tsx

```diff
@@ -237,24 +197,23 @@ export const Tabs = ({
         ref={headerRef}
       >
         <ul className={styles['tabs__list']} role="tablist">
-          {getTabs().map((tab, i) => {
+          {tabs.map((tab, i) => {
             const isActive = activeIndexState === i;
             return (
               <li
                 className={clsx(
                   styles['tabs__item'],
                   isActive && styles['eds-is-active'],
                 )}
-                key={'tabs-item-' + i}
```

Comment:
Best practice is to use a unique identifier for `key`, not the loop index.

https://reactjs.org/docs/lists-and-keys.html